pages/en-us/components/icons.mdx

```diff
@@ -0,0 +1,72 @@
+import { Layout, Playground, Attributes } from 'lib/components'
+import Title from 'lib/components/playground/title'
+import { Spacer } from 'components'
+import * as Icon from '@zeit-ui/react-icons'
+import Github from '@zeit-ui/react-icons/github'
+import IconsGallery from 'lib/components/displays/icons'
+
+export const meta = {
+  title: 'Icons',
+  group: 'General',
+}
+
+## Icons
+
+Display semantic vector graphics in SVG format.
+
+<Title title="Installation" desc="Install the package in your project directory." />
+
+```
+// with yarn
+yarn add @zeit-ui/react-icons
+
+// with npm
+npm install @zeit-ui/react-icons
```

Comment:
Or can we try the `Tab` component? like [this](https://react.zeit-ui.co/en-us/guide/installation)